packages/mds-agency/request-handlers.ts

```diff
@@ -230,7 +230,7 @@ export const submitVehicleEvent = async (req: AgencyApiRequest, res: AgencySubmi
   try {
     validateEvent(event)
   } catch (err) {
-    logger.info(`Event ValidationError for ${providerName(provider_id)}. Error: ${err}`)
+    logger.info(`Event ValidationError for ${providerName(provider_id)}. Error: ${JSON.stringify(err)}`)
```

Comment:
Feels like there's room for some kind of helper function, e.g. `logger.info(printErrorMessage(provider_id, err))` but that's a minor nit.

Reply:
Yeah, this isn't meant to last for a long time though. We wanted to quantify how much data is getting past our hand-made validators compared to the validation imposed by a JOI schema, so it will probably be decently reworked at some point soon.